envs/monkey_zoo/packer/setup_mimikatz_15.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Create a new user
```

Comment:
These names are not consistent. Either way, this looks like it creates an image in GCP, not user